environ.py

```diff
@@ -0,0 +1,29 @@
+# standard libraries
+import json, logging, os
+
+# entry-level job modules need to be one-level beneath root
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+logger = logging.getLogger(__name__)
+
+
+# Set up ENV
+
+config_path = os.path.join(ROOT_DIR, os.getenv('ENV_PATH', os.path.join('config', 'secrets', 'env.json')))
+
+try:
+    with open(config_path) as env_file:
+        ENV = json.loads(env_file.read())
+except FileNotFoundError:
+    logger.error('Configuration file could not be found; please add env.json to the config directory.')
+    ENV = None
+
+logging.basicConfig(level=ENV['LOG_LEVEL'])
+
+# Add ENV key-value pairs to environment, skipping if the key is already set
+logger.debug(os.environ)
+for key, value in ENV.items():
```

Comment:
Cool, I think this is the fix for MyLA https://github.com/tl-its-umich-edu/my-learning-analytics/issues/787